app/design/adminhtml/default/default/template/he_twofactor/google/auth.phtml

```diff
@@ -96,7 +96,7 @@
 
                 <div class="clear"></div>
                 <div class="form-buttons">
-                    <a class="left" href="<?php echo $this->getUrl('adminhtml', array('_nosecret' => true)); ?>">&laquo; <?php echo Mage::helper('adminhtml')->__('Back to Login'); ?></a>
+                    <a class="left" href="<?php echo $this->getUrl('adminhtml/index/logout'); ?>">&laquo; <?php echo Mage::helper('adminhtml')->__('Back to Login'); ?></a>
```

Comment:
"‹ Back to Login" doesn't indicate to the user that they are being logged out, though I guess it's implied. Would it make sense to change this to "Log Out" or something like that for clarity?